mtp_noms_ops/templates/security/accept_or_reject_check.html

```diff
@@ -152,38 +164,44 @@ <h3 class="heading-medium">{% trans 'Credit decisions matching debit card or pri
       </table>
     </div>
 
-    {% if check.status == 'pending' %}
-
-      <h2 class="heading-medium">{% trans 'Your decision' %}</h2>
-      <div class="form-group">
-        {% include 'mtp_common/forms/textarea.html' with field=form.decision_reason input_classes='form-control-3-4' %}
-        <button type="submit" class="button" name="fiu_action" value="accept">
-          {% trans 'Accept credit' %}
-        </button>
-        <button type="submit" class="button button-warning" name="fiu_action" value="reject">
-          {% trans 'Reject credit' %}
-        </button>
-      </div>
-
-    {% else %}
-
-      <h2 class="heading-medium">{% trans 'Decision' %}</h2>
-      <p>
-        {% blocktrans trimmed with resolved=check.status|format_security_check_status|lower user=check.actioned_by_name %}
-          This credit was {{ resolved }} by {{ user }}.
-        {% endblocktrans %}
-      </p>
-      <p>
-        <strong>{% trans 'Decision details' %}:</strong>
-        {% if check.decision_reason %}
-          {{ check.decision_reason }}
-        {% else %}
-          {% trans 'No decision reason entered' %}
-        {% endif %}
-      </p>
-
-    {% endif %}
-
-  </form>
+    <form method="post">
+      {% csrf_token %}
+      {% include 'mtp_common/forms/error-summary.html' with form=form only %}
+
+      {% if check.status == 'pending' %}
+
+        <div id=accept-or-reject-form class="form-group">
```

Comment:
it's probably better to quote attributes. is this id referenced anywhere?

Reply:
Ah, yes, we need to jump down to this point (or on second thoughts early up the page) so the error messages are in view, since we moved the form tag down the page.

Reply:
We've adding in the url fragment to target the form should it error now, and this id handles the page loading so error messages and form are in page view.